src/shared-components/tabs/test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { TabItem } from './style';
+
+import Tabs from './index';
+
+describe('<Tabs />', () => {
+  describe('default props', () => {
+    test('renders the correct props', () => {
+      const wrapper = shallow(
+        <Tabs
+          activeTabId={1}
+          tabItems={[
+            { id: 1, text: 'Rain Drop' },
```

Comment:
lol we can probably update the text to be more normal 😆

Reply:
done